mmv1/products/redis/api.yaml

```diff
@@ -400,3 +400,51 @@ objects:
           an existing instance. For DIRECT_PEERING mode value must be a CIDR range of size /28, or
           "auto". For PRIVATE_SERVICE_ACCESS mode value must be the name of an allocated address 
           range associated with the private service access connection, or "auto".
+      - !ruby/object:Api::Type::NestedObject
+        name: persistenceConfig
```

Comment:
Ah, we probably have to use `default_from_api` anyways, on this field. The API returns the block by default, causing a diff. We can keep the default for the subfield, though.